libs/designsystem/src/lib/components/list/list.component.ts

```diff
@@ -200,7 +200,7 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
 
   ngOnChanges(): void {
     this._isSectionsEnabled = !!this.getSectionName;
-    if (this.items.length === 0) return;
+    if (!this.items || this.items.length === 0) return;
```

Comment:
Did we upgrade typescript to be able to use the `.?` property selector which I can't remember what is called right now? If so - consider using that 😄